File: Npc/11003391.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11003391: Evagor
/// </summary>
public class _11003391 : NpcScript {
    protected override int First() {
        return 10;
    }

    // Select 0:
    // $script:1122180007011902$
    // - You should stay put, stranger.
    protected override int Select() => 0;

    protected override int Execute(int selection) {
        switch (Id, Index++) {
            case (10, 0):
                // $script:1122180007011903$
                // - You should stay put, stranger.
                return -1;
        }

        return default;
    }

    protected override NpcTalkButton Button() {
        return (Id, Index) switch {
            (10, 0) => NpcTalkButton.Close,
            _ => NpcTalkButton.None,
        };
    }
}

File: Npc/11003391.py
""" 11003391: Evagor """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 10

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:1122180007011902$
        # - You should stay put, stranger.
        return None # TODO

    def __10(self, pick: int) -> int:
        # $script:1122180007011903$
        # - You should stay put, stranger.
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (10, 0):
            return Option.CLOSE
        return Option.NONE
